lib/components/PageLoader.js

```diff
@@ -8,7 +8,7 @@ import { PATH_VARIANTS, ROTATE_VARIANTS } from "../constants/pageLoader";
 
 const PageLoader = ({ text = "", ...otherProps }) => {
   return (
-    <div className="neeto-ui-pageloader__wrapper" {...otherProps}>
+    <div className="neeto-ui-pageloader__wrapper" data-testid="pageloader" {...otherProps} >
```

Comment:
@prasidhprabudhan _a As discussed lets pass the data-testid from the test itself in this case.